docs/multicluster/user-guide.md

```diff
@@ -81,6 +81,12 @@ antrea-agent.conf: |
     namespace: "" # Change to the Namespace where antrea-mc-controller is deployed.
 ```
 
+The `enableGateway` can be disabled only when Pod-to-Pod direct connectivity already exists
```

Comment:
This suggests that `enableGateway` is true by default, which is not the case?
It should be:
> In order for Multi-cluster features to work, `enableGateway` should be set to true by the user, except when Pod-to-Pod direct connectivity already exists (e.g., provided by the cloud provider) and `endpointIPType` is configured as `PodIP`.

Reply:
Updated, thanks.